lib/custom_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class CustomHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
+  def image(*args)
+    %(<a href="#{args.first}" target="_blank">#{super}</a>)
```

Comment:
I don't think it's worth blocking anything over the new tab question btw. Merge first ask questions later I think.